src/ExtEventLoop.php

```diff
@@ -178,8 +178,7 @@ public function tick()
 
         $this->futureTickQueue->tick();
 
-        // @-suppression: https://github.com/reactphp/react/pull/234#discussion-diff-7759616R226
-        @$this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
+        $this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
```

Comment:
That is weired 😕

Interestingly, it looks like the following code hides **only** the warnings from ext-event:

```php
$loop = new React\EventLoop\ExtEventLoop();

set_error_handler(function () {
    return false; // <-- Just pass through to the standard PHP error handler
});

$loop->addTimer(0.001, function () {
    echo $undefined;
});

$loop->addTimer(0.001, function () {
    throw new Exception("Exception inside my callback");
});

$loop->run();
restore_error_handler();
```

Produces just:

```
PHP Notice:  Undefined variable: undefined in /Users/jsor/Code/~react/event-loop/test.php on line 12

PHP Fatal error:  Uncaught exception 'Exception' with message 'Exception inside my callback' in /Users/jsor/Code/~react/event-loop/test.php:16
```

Without the error handler, the ext-event warning is also shown:

```
PHP Fatal error:  Uncaught exception 'Exception' with message 'Exception inside my callback' in /Users/jsor/Code/~react/event-loop/test.php:16

PHP Warning:  EventBase::loop(): An error occurred while invoking the callback in /Users/jsor/Code/~react/event-loop/src/ExtEventLoop.php on line 191

PHP Fatal error:  Uncaught exception 'Exception' with message 'Exception inside my callback' in /Users/jsor/Code/~react/event-loop/test.php:16
```